test/testallfilesdeleted.cpp

```diff
@@ -300,8 +300,10 @@ private slots:
     }
 
     void testSelectiveSyncNoPopup() {
+        auto original = FileInfo::A12_B12_C12_S12();
```

Comment:
I understand the original is not to be modified, so maybe const?

Reply:
done